main.less

```diff
@@ -30,16 +25,24 @@
     border: 1px solid #deca7e;
 }
 
-.errors {
-    width: 25px;
-    cursor: pointer;
-}
+.hint-marker-positioning {
+  left: auto !important; /* CodeMirror will set this value, but we don't want it set! */
+  right: 1em;
+  background-color: transparent;
 
-.gutterCursor {
-    cursor: pointer;
+  margin: 0px;
+  padding: 0px;
```

Comment:
I think thats just css I stole from thimble's original htmlhinter, I'm not really great at css so I didn't bother touching it